Add macOS URL support for more browsers

wrgoto opened the pull request:
Adds URL support for other Chrome-based browsers on macOS:

- Brave Browser Beta
- Sidekick
- Opera

sindresorhus commented:
Can you add beta/canary versions of the existing browsers there too? While you're at it.

sindresorhus commented:
You also need to update the docs (readme/index.d.ts).

sindresorhus commented on index.d.ts:
```diff
@@ -74,7 +74,7 @@ declare namespace activeWindow {
 
 		/**
-		URL of the active browser tab if the active window is Safari, Chrome, Edge, or Brave.
+		URL of the active browser tab if the active window is Safari, Chrome (includes Beta, Dev, and Canary), Edge (includes Beta, Dev, and Canary), Brave (includes Beta and Nightly), or Opera.
```
Missing `Sidekick`.

And needs to be updated in the readme too.

wrgoto commented:
Rebased with `main` and added support for beta, dev, and canary. Full list of added browsers

- Brave Browser (Beta and Nightly)
- Google Chrome (Beta, Dev, and Canary)
- Microsoft Edge (Beta, Dev, and Canary)
- Sidekick
- Opera

Brave claims they have a development build, but their site redirects to the beta build.

wrgoto commented:
Additionally, Opera uses the same app name string for its beta, dev, and canary builds. So you if you have any of them it'll work the same.
